solana/program_v2/packages/tests/src/pass-suite/change_pass_gatekeeper.test.ts

```diff
@@ -70,10 +70,12 @@ describe('Change pass gatekeeper', () => {
         subject.publicKey,
         TOKEN_PROGRAM_ID,
```

Comment:
We actually don't need to pass this in as it's always going to be the TOKEN_PROGRAM_ID (i.e. the same token program we check on chain)...